pkg/runtime/wfengine/backends/actors/workflowstate.go

```diff
@@ -127,6 +129,14 @@ func (s *workflowState) GetSaveRequest(actorID string) (*actors.TransactionalReq
 		return nil, err
 	}
 
+	// if s.workflowStartTime is not nil then append req.operations with a new operation to update the workflowStartTime
+	if !s.workflowStartTime.IsZero() {
```

Comment:
This adds one more round-trip to the DB per each request. The performance impact is non-trivial, and I'm not sure it's worth for a metric?

Can we find another way that doesn't require writing and loading a new state key?

Reply:
Agree, from both a performance and cost perspective this isn't desirable